src/vivarium/framework/time.py

```diff
@@ -150,6 +162,13 @@ def step_forward(self, index: pd.Index) -> None:
             clocks_to_update = self._individual_clocks.get(update_index)
             if not clocks_to_update.empty:
                 clocks_to_update["step_size"] = self._step_size_pipeline(update_index)
+                # Simulants that were flagged to get moved to the end should have a next event time
+                # of stop time + 1 minimum timestep
+                clocks_to_update.loc[self._simulants_to_untrack, "step_size"] = (
+                    self.stop_time + self.minimum_step_size - self.time
```

Comment:
why do you need to add the minimum step size? Just to ensure they really go off the end?

Reply:
yeah, long story short the end emitter in the `finalize` part of the simulation acts during the stop time, and requires a step size. If we don't add the minimum timestep here, `finalize` reads in a step size of 0 and errors out.